### PR TITLE
translated git bisect in english

### PR DESCRIPTION
**Pull request by GusTTShowbiz**

### CODE REVIEW

**Comment by ElevenWilson** [comment]
7 words might be misspelled, please check them: jpg, phpunit, rpierlot, seach, SVN, ThatWontHappenAgainISwearTest, Yay

File: _posts/2017-11-03-git-bisect.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+categories:
+    - git
+tags:
+    - amp
+    - git
+    - php
+cover: /assets/2017-10-26-debugging-with-git/branching-illustration.png
+---
+
+When deploying new features to a production environment, risks of regression exist. Especially on big applications with a lot of non-tested code.
```
Comment (snroki) [inline _posts/2017-11-03-git-bisect.md]: on or in big applications ?

File: _posts/2017-11-03-git-bisect.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+We are never safe from negative behaviour, or impacts that need our immediate attention. 
+
+In a project using Git as a source code management tool, it appears essential to rapidly find the source of our problem. 
+When dealing with large teams composed of many people, every deployment comes with a lot of commits.
+But if the origin of the problem is hidden in the middle of all those commits, it is quite complicated to check very commit for the source of the issue.
```
Comment (snroki) [inline _posts/2017-11-03-git-bisect.md]: very => every

File: _posts/2017-11-03-git-bisect.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+---
+
+When deploying new features to a production environment, risks of regression exist. Especially on big applications with a lot of non-tested code.
+We are never safe from negative behaviour, or impacts that need our immediate attention. 
+
+In a project using Git as a source code management tool, it appears essential to rapidly find the source of our problem. 
+When dealing with large teams composed of many people, every deployment comes with a lot of commits.
+But if the origin of the problem is hidden in the middle of all those commits, it is quite complicated to check very commit for the source of the issue. 
+
+Git is very famous tool in software development, and it comes with a bunch off commands that prevent us from missing good old SVN.
```
Comment (snroki) [inline _posts/2017-11-03-git-bisect.md]: a very famous no ?

File: _posts/2017-11-03-git-bisect.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+The `bisect` command allows you to automate the search process. 
+Indeed, if you have the luxury of adding a test that highlights the bug, you can launch git bisect as follow: 
+```
+git bisect run vendor/bin/phpunit --filter ThatWontHappenAgainISwearTest
+```
+The command `vendor/bin/phpunit --filter ThatWontHappenAgainISwearTest` will be launched at every step seen before, and you'll automatically have the hash of the commit you're looking for.
+
+## Recap 
+
+The use of `git bisect` allows us to rapidly debug, isolating a bad commit with a binary search.
```
Comment (snroki) [inline _posts/2017-11-03-git-bisect.md]: isolate no ? (it allows us to isolate)

**Comment by ElevenWilson** [comment]
6 words might be misspelled, please check them: jpg, phpunit, rpierlot, SVN, ThatWontHappenAgainISwearTest, Yay

File: _posts/2017-11-03-git-bisect.md (hunk 3 above)
Comment (GusTTShowbiz) [reply _posts/2017-11-03-git-bisect.md]: yes

File: _posts/2017-11-03-git-bisect.md (hunk 2 above)
Comment (GusTTShowbiz) [reply _posts/2017-11-03-git-bisect.md]: yes

File: _posts/2017-11-03-git-bisect.md (hunk 1 above)
Comment (GusTTShowbiz) [reply _posts/2017-11-03-git-bisect.md]: yes

**Comment by ElevenWilson** [comment]
6 words might be misspelled, please check them: jpg, phpunit, rpierlot, SVN, ThatWontHappenAgainISwearTest, Yay

**Comment by ElevenWilson** [comment]
7 words might be misspelled, please check them: functionalities, jpg, phpunit, rpierlot, SVN, ThatWontHappenAgainISwearTest, Yay

**Comment by ElevenWilson** [comment]
6 words might be misspelled, please check them: jpg, phpunit, rpierlot, SVN, ThatWontHappenAgainISwearTest, Yay

**Comment by ElevenWilson** [comment]
6 words might be misspelled, please check them: jpg, phpunit, rpierlot, SVN, ThatWontHappenAgainISwearTest, Yay

**Pull request merged**